lib/jxl/encode.cc

```diff
@@ -1588,6 +1634,53 @@ JxlEncoderStatus JxlEncoderAddJPEGFrame(
   return JXL_ENC_SUCCESS;
 }
 
+static bool CanDoFastLossless(const JxlEncoderFrameSettings* frame_settings,
+                              const JxlPixelFormat* pixel_format,
+                              bool has_alpha) {
+  if (!frame_settings->values.lossless) {
+    return false;
+  }
+  if (frame_settings->values.frame_index_box) {
+    return false;
```

Comment:
Maybe put a TODO here because I don't think there's an inherent reason why this combination cannot work.

Reply:
I put a general TODO in the function; it should be relatively easy to add support for a few of those, follow-up PRs are welcome ;)